integrations/aws-v3/aws/core/interfaces/action.py

```diff
@@ -1,54 +1,33 @@
-from typing import Dict, Any, List, Union
+from typing import Dict, Any, List
 from abc import ABC, abstractmethod
 from aiobotocore.client import AioBaseClient
 from typing import Type, Protocol
-import asyncio
+from loguru import logger
 
 
 class Action(ABC):
-    """Base interface for actions that support single execution"""
 
     def __init__(self, client: AioBaseClient) -> None:
-        self.client: AioBaseClient = client
-
-    async def execute(self, identifier: Any) -> Dict[str, Any]:
-        response = await self._execute(identifier)
-        return response
-
-    async def execute_for_identifiers(
-        self, identifiers: List[str]
-    ) -> List[Dict[str, Any]]:
-        """Execute action for multiple identifiers by calling execute for each one."""
-        return await asyncio.gather(
-            *[self.execute(identifier) for identifier in identifiers]
+        """aiobotocore's concrete clients provide methods like `get_bucket_tagging`
+        at runtime, but these are not declared on `AioBaseClient`'s static types.
+        Using `Any` here avoids per-call type ignores while keeping runtime behavior
+        unchanged.
+        """
+        self.client: Any = client
+
+    async def execute(self, identifiers: List[Any]) -> List[Dict[str, Any]]:
+        logger.info(
```

Comment:
If an action will run per resource (i.e a single ARN) can we conver this log to debug?
since I think this will bombard us in the logs